src/components/shared/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { Card } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import Rating from '../Rating/Rating';
+import { Product } from '../../../types/product';
+
+const ProductCard :FC<ProductProps> = ({ productData }:ProductProps) => {
+  const {
+    id, name, imageURL, price, rating, numReviews,
+  } = productData;
+  return (
+    <Card className="my-3 rounded">
+      <Link to={`/product/${id}`}>
+        <Card.Img src={`${process.env.REACT_APP_BACKEND_URL}${imageURL}`} variant="top" />
```

Comment:
why do not u use the link sent in the response instead, i know it is the same but for scalability sake, i believe it is gonna be much better to use the link sent to u in the response as if we are talking about a real app, we may upload our images somewhere else rather than our own, besides in such apps images supposed to have more than one sample, so following the response from my POV